bundles/org.openhab.ui/web/src/assets/definitions/widgets/system/index.js

```diff
@@ -2,13 +2,15 @@ import { WidgetDefinition, pt, pb } from '../helpers.js'
 import { actionGroup, actionParams } from '../actions.js'
 
 const VariableParameter = pt('variable', 'Variable', 'Name of the variable to set on input change')
+const VariableKeyParameter = pt('variableKey', 'Variable Key', 'Consider the variable value is an object and set the corresponding deep property within that object using a key syntax. Examples: <code>user.name</code>, <code>user[0].address[1].street</code>, <code>[0]</code>, <code>[0].label</code>. The inner property and its parent hierarchy will be created if missing.').a()
 const ClearVariableParameter = pb('clearVariable', 'Clear Variable After Action', 'Name of the variable to clear after performing the action')
+const ClearVariableKeyParameter = pb('clearVariableKey', 'Clear Key Propertoy Inside An Object After Action', 'Consider the variable value is an object and remove a corresponding deep property within that object using a key syntax. Examples: <code>user.name</code>, <code>user[0].address[1].street</code>, <code>[0]</code>, <code>[0].label</code>. The property will be removed after performing the action.').a()
```

Comment:
```suggestion
const ClearVariableKeyParameter = pb('clearVariableKey', 'Clear Object Variable Property After Action', 'Consider the variable value is an object and remove a corresponding deep property within that object using a key syntax. Examples: <code>user.name</code>, <code>user[0].address[1].street</code>, <code>[0]</code>, <code>[0].label</code>. The property will be removed after performing the action.').a()
```

This should update the related docs and solve the remarks in the process after you run the generate.js script.